### ISSUE TITLE
Highlight comments that mention my name

### ISSUE THREAD

**Issue by ttencate** [mkalam-alami/ludumdare-feedback-friends#2]
If someone specifically replies to one of my comments, especially if it's not on my own game but on theirs, the actual LD site makes no effort to tell me about this. Can we do better?

Username parsing will have to be a bit lenient because many people have spaces and/or punctuation in their name. And to avoid false positives, it should probably only give results for games that I actually commented on.

**Comment by mkalam-alami** [comment]
This is an interesting idea.

Maybe we could simply support @mentions? If we make it clear on the entry page that we support them (like by showing them in a 3rd column, with a short explaination or sthg) people could know they can use the @-based syntax.

**Comment by mkalam-alami** [comment]
I just gave it a try, on the LD35 entries it already yields some comments ([example](http://feedback.ld.intricati.com/?event=ludum-dare-35&uid=9172)), so it does look useful!

The implementation is not perfect though:

* For more complex author names like "Frozen Fractal" people might be tempted to remove the space ([your entry page](http://feedback.ld.intricati.com/?event=ludum-dare-35&uid=7882) does miss a comment because of that)
* If a user is called, say "WanXXX" it will be spammed all mentions for "Wan" ¯\_(ツ)_/¯

...but I guess we could live with that!

**Comment by ttencate** [comment]
Sweet! Perhaps we should just match on letters only. Or try harder if there's an @ sign in front. In any case, I think the mentions should be limited to other people who commented on that thread -- that will make WanXXX happy :)

Sorry I haven't been much help beyond commenting yet. I'll try to make some time this weekend to contribute some actual code.

**Comment by mkalam-alami** [comment]
> I guess you're currently triggering on that exclusively?

Yes I tried to keep the query simple for performance, to make things clear I added a small help if there's no mentions yet on an entry ([example](http://feedback.ld.intricati.com/?event=minild-69&uid=68143)). Though your idea of restricting mentions to other commenters shouldn't be too expensive, it could work!

And don't worry, there's no rush at all improving this site. I'm just having fun trying stuff since I have some free time these days ;)
